src/common/containers/Home.tsx

```diff
@@ -118,6 +125,10 @@ class Home extends React.Component<HomeProps> {
       </div>
     </PaperButton>
   )
+
+  private handleClickNewHere = () => {
+    RouterDialog.openDialog(this.props.history, 'new-here');
```

Comment:
Why does this need to have a route? Are we going to link people to `/new-here`?

Reply:
I think it would be reasonable to have the option of linking to `//new-here`.